plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/pattern-screen/categories.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+// @todo: Implement the right copy for the pattern categories: https://github.com/woocommerce/woocommerce/issues/48381
+export const PATTERN_CATEGORIES = {
```

Comment:
Can we make a change here to ensure the categories are based on all patterns fetched via the PTK endpoint instead of having them hardcoded this way?

Reply:
Given that this looks like a broader discussion (including some changes on the PKT API), what do you think if I open a dedicated issue?

Reply:
> what do you think if I open a dedicated issue?

sounds good 👍

Reply:
https://github.com/woocommerce/woocommerce/issues/48381